msteams-platform/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md

```diff
@@ -69,12 +72,17 @@ This provides an engaging narrative highlighting your solution's main features,
 > Make sure to precisely copy the descriptions you wrote in your AppSource entry to your manifest — the values must match. Teams will only use the descriptions you provide in the manifest.
 
 **Do's:**
+* Listing features as bullet points will aid readers scanning your description.  
+
+
+> [!NOTE]
+>To create bulleted list, below formatting must be used. No other formatting is allowed currently for long description.   
```

Comment:
Since we cannot confirm whether this is true, should this information be deleted?